File: java/warmup-1/posNeg.java
/* Given 2 int values, return true if one is negative and one is positive. 
 * Except if the parameter "negative" is true, then return true only if both 
 * are negative.
 */
public boolean posNeg(int a, int b, boolean negative) {
    if(negative)
        return a < 0 && b < 0;
    else
        return a * b < 0;
}

File: python/warmup-1/pos_neg.py
# Given 2 int values, return True if one is negative and one is positive. 
# Except if the parameter "negative" is True, then return True only if both 
# are negative.
def pos_neg(a, b, negative):
    if negative:
        return a < 0 and b < 0
          
    return a * b < 0
